lib/netsuite/records/opportunity_item_list.rb

```diff
@@ -0,0 +1,31 @@
+module NetSuite
+  module Records
+    class OpportunityItemList
```

Comment:
This should inherit from `Support::Sublist`. Checkout this sublist as an example:

https://github.com/NetSweet/netsuite/blob/master/lib/netsuite/records/invoice_item_list.rb